### PR TITLE
[Term Entry] General: Semantic HTML

### PR DESCRIPTION
**Pull request by caupolicandiaz**
### Description

Addition of a general entry on semantic html

### Type of Change

- Adding a new entry

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@caupolicandiaz Looks good

File: content/general/concepts/semantic-html/semantic-html.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+**Semantic HTML** is the use of HTML tags that are representative of the content, so that browsers can appropriately parse the content of a site. This is beneficial as it ensures that content can be described accurately for those that rely on accessibility features.
+
+## Non-Semantic Tags
+
+Tags such as a `<div>` or `<span>` represent non-semantic HTML: these tags do not convey any information about their content. These tags are multipurpose containers that can support a variety of content. The use of these tags obscures the contents of a page and does not capitalize on the inbuilt attributes available in the use of the appropriate semantic tags.
+
+## Semantic Tags
+
+Semantic tags are those that are designated for a specific purpose such as `<header>` or `<figure>`, which is communicated by the tag name. In addition to providing better clarity to the organization and content of a page semantic tags enable additional features for accessibility. For example, the use of the `<button>` tag for controlling any embedded media also allows for navigation, and use of the button via keyboard.
```
Comment (HishamT) [inline content/general/concepts/semantic-html/semantic-html.md:25]:
```suggestion
Semantic tags, such as `<header>` or `<figure>`, are designated for a specific purpose which is communicated by the tag name. In addition to providing better clarity to the organization and content of a page, semantic tags enable additional features for accessibility. For example, using the `<button>` tag for controlling any embedded media also allows for navigation, and use of the button via keyboard.
```

**Review by HishamT — COMMENTED** [review]
Hey @caupolicandiaz, I reviewed this entry and made some suggestions. :grin:

**Comment by caupolicandiaz** [comment]
@HishamT I've incorporated your notes. Thanks

**Review by KTom101 — APPROVED** [review]
@caupolicandiaz LGTM, too. Just need to change the meta-description so the apostrophe (') in "site's" no longer causes an error with the checks. 🙂

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz LGTM, too. Just need to change the meta-description so the apostrophe (') in "site's" no longer causes an error with the checks. 🙂

Thanks, I was wondering what I did.

**Comment by KTom101** [comment]
Live link: https://www.codecademy.com/resources/docs/general/semantic-html

**Pull request merged**